lib/ransack_ui/ransack_overrides/helpers/form_builder.rb

```diff
@@ -141,9 +141,9 @@ def attribute_collection_for_base(base, attributes=nil)
 
           # Set model name as label for 'id' column on that model's table.
           if c == 'id'
-            foreign_klass = object.context.traverse(base).model_name
+            foreign_klass = object.context.traverse(base)
             # Check that model can autocomplete. If not, skip this id column.
-            next nil unless foreign_klass.constantize._ransack_autocompletes_through.present?
+            next nil unless foreign_klass._ransack_autocompletes_through.present?
```

Comment:
Autocomplete should be `nil` when not set, so shouldn't need to also call `.present?`